index.html

```diff
@@ -366,7 +366,7 @@ <h3 id="w3c-coordination">W3C Groups</h3>
 
         <dt><a href="https://www.w3.org/community/wicg/">Web Platform Incubator Community Group</a></dt>
         <dd>
-          This group provides a lightweight venue for proposing, incubating and discussing new web platform features. Several proposals related to Web Performance were moved there: <a href="https://w3c.github.io/frame-timing/">Frame Timing</a>, <a href="https://w3c.github.io/server-timing/">Server Timing</a>, <a href="https://w3c.github.io/reporting/">Reporting</a>, and <a href="https://w3c.github.io/network-error-logging/">Network Error Logging</a>.
+          This group provides a lightweight venue for proposing, incubating and discussing new web platform features. The Web Performance Working group will incubate and review new proposals that are within scope of our charter within the WICG. Once such WICG-incubated proposal has wide support and is available in at least one major browser, it may be adopted by the Web Performance Working group.
```

Comment:
Did we conclude on one or two? Shouldn't we have a clear sign of interest from more than one before adopting a draft in the WG.

Reply:
So far we've said and enforced "one shipping" plus wider support from the group, which I personally interpret as at least one other vendor expressing interesting and support. That said, it may be worth spelling this out more explicitly.

> Once such WICG-incubated proposal is implemented and available in at least one major browser, and has support from one more, it may be adopted by the Web Performance Working group.

WDYT?

Reply:
I like your rewording. Much better imho.